mopub-sdk-base-android-binding/Transforms/Metadata.xml

```diff
@@ -21,32 +21,74 @@
     <method name="remove" return="void" abstract="false" native="false" synchronized="false" static="false" final="false" deprecated="not deprecated" visibility="public">
     </method>
   </add-node>
-  
+
+  <add-node path="/api/package[@name='com.mopub.common.privacy']/class[@name='SyncRequest']">
+    <method name="compareTo" return="int" abstract="false" native="false" synchronized="false" static="false" final="false" deprecated="not deprecated" visibility="public">
+      <parameter name="o" type="java.lang.Object" />
+    </method>
+  </add-node>
```

Comment:
I won't pretend I know exactly what these `add-node`s are doing... My guess would be adding to the API something that wasn't there from the automatic binding? Can you maybe elaborate on what these are doing?

Reply:
I will not pretend either, the documentation is super vague on this one. I just had the exact same issue with `compareTo` before, used the same solution as before and it worked.